server/http_config_handler.go

```diff
@@ -116,11 +81,147 @@ func productsHandler(w http.ResponseWriter, r *http.Request) {
 	jsonResult(w, res)
 }
 
-// testHandler tests a configuration by class
-func testHandler(w http.ResponseWriter, r *http.Request) {
+func devicesConfig[T any](h config.Handler[T]) []map[string]any {
+	var res []map[string]any
+
+	// omit name from config
+	for _, dev := range h.Devices() {
+		conf := dev.Config()
+
+		dc := map[string]any{
+			"name": conf.Name,
+			"type": conf.Type,
+		}
+
+		if configurable, ok := dev.(config.ConfigurableDevice[T]); ok {
+			// from database
+			dc["id"] = configurable.ID()
+			dc["config"] = conf.Other
```

Comment:
Das scheint noch falsch zu sein. Die Config Property ist in der Antwort immer ein leeres Objekt.

```
{
"config": {},
"id": 1,
"name": "db:1",
"type": "template"
}
```

Reply:
Anmerkung: Nach einem Neustart sind die Werte korrekt vorhanden. Wir wollen den User aber nicht nach jedem Bearbeitungsschritt den Server neu Starten lassen. Im vorherigen Stand war es schon so, dass das funktioniert hatte.

Reply:
Passiert also nur bei neu erzeugten?

Reply:
Ja